tests/features/event/test_event.py

```diff
@@ -56,13 +56,15 @@ def setUp(self):
         self.event = self.application.make("event")
         self.event.listen(UserAddedEvent, [SendEmailListener])
 
+    def tearDown(self):
```

Comment:
Here I am resetting events that are listened to have idempotent tests and to avoid having those events in other unit tests.